<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1439</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="11" column="6">
     <widget class="QDoubleSpinBox" name="inputX">
      <property name="locale">
       <locale language="C" country="AnyTerritory"/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-99.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="vertCountLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Vertices count</string>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QPushButton" name="moveButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Move object</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Transform</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="settingsButton">
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QSpinBox" name="edgeCount">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
    </item>
    <item row="15" column="6">
     <widget class="QDoubleSpinBox" name="inputZ">
      <property name="locale">
       <locale language="C" country="AnyTerritory"/>
      </property>
      <property name="minimum">
       <double>-99.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QSpinBox" name="vertCount">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
    </item>
    <item row="12" column="6">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QListView" name="listView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QPushButton" name="rotateButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Rotate object</string>
      </property>
     </widget>
    </item>
    <item row="10" column="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="6">
     <widget class="QDoubleSpinBox" name="inputY">
      <property name="locale">
       <locale language="C" country="AnyTerritory"/>
      </property>
      <property name="minimum">
       <double>-99.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QPushButton" name="scaleButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Scale object</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="edgeCountLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Edge count</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="17" colspan="6">
     <widget class="s21::Drawer" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="14" column="6">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1439</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionImage"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionChangeSettings"/>
   </widget>
   <widget class="QMenu" name="menuGif">
    <property name="title">
     <string>Record gif</string>
    </property>
    <addaction name="actionStartGif"/>
    <addaction name="actionStopGif"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuGif"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionGif">
   <property name="text">
    <string>Record GIF</string>
   </property>
  </action>
  <action name="actionProjection_type">
   <property name="text">
    <string>Load Settings</string>
   </property>
  </action>
  <action name="actionChangeSettings">
   <property name="text">
    <string>Change settings</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionStartGif">
   <property name="text">
    <string>Start Record</string>
   </property>
  </action>
  <action name="actionStopGif">
   <property name="text">
    <string>Stop Record</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>s21::Drawer</class>
   <extends>QOpenGLWidget</extends>
   <header>Model/drawer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
